resources/assets/js/annotations/components/annotationCanvas/labelBot.vue

```diff
@@ -2,6 +2,8 @@
 import LabelbotIndicator from '../labelbotIndicator.vue';
 import LabelbotPopup from '../labelbotPopup.vue';
 import { LABELBOT_STATES } from '../../mixins/labelbot.vue';
+import { makeBlob } from '../../makeBlobFromCanvas.js';
```

Comment:
I think it's ok to put this into `utils.js` too, so we don't have an extra file for this.